Makefile

```diff
@@ -88,51 +85,52 @@ NO_RE2C_BISON_PREFIX := -no-re2c-bison
 NO_TESTS_PREFIX := -no-tests
 
 ifeq ($(USE_NINJA),1)
-BUILD := ninja
+BUILD_CMD := ninja
 BUILD_FILE := build.ninja
 GENERATOR := Ninja
 else
-BUILD := $(MAKE) --no-print-directory
+BUILD_CMD := $(MAKE) --no-print-directory
 BUILD_FILE := Makefile
 GENERATOR := "Unix Makefiles"
 endif
 
 CMAKE_DIR = out/$($(1)_DIR)$($(2)_DIR)$($(3)_DIR)
-EXE_TARGET = $($(1)_PREFIX)$($(2)_PREFIX)$($(3)_PREFIX)
+BUILD_TARGET = $($(1)_PREFIX)$($(2)_PREFIX)$($(3)_PREFIX)
+INSTALL_TARGET = install-$($(1)_PREFIX)$($(2)_PREFIX)$($(3)_PREFIX)
 TEST_TARGET = test-$($(1)_PREFIX)$($(2)_PREFIX)$($(3)_PREFIX)
 
-define DEFAULT
-.PHONY: $(3)$($(4)_PREFIX) test$($(4)_PREFIX)
-$(3)$($(4)_PREFIX): $(call EXE_TARGET,$(1),$(2),$(4))
-	ln -sf ../$(call CMAKE_DIR,$(1),$(2),$(4))$(3) out/$(3)$($(4)_PREFIX)
```

Comment:
Yeah, I've noticed this too. I guess the README should be updated, and the test runner should use out/clang/Debug too.